### ISSUE TITLE
[pigeon] 0.1.13 Windows only supports file:* and data:* URIs

### ISSUE THREAD

**Issue by creativecreatorormaybenot** [flutter/flutter#68583]
## Steps to Reproduce

@gaaclarke I tried to run `pigeon: 0.1.13` after your https://github.com/flutter/packages/pull/230 fix PR. However, the issue still appears for me - now just a little bit different 😅

```
D:\creativecreatorormaybenot\microphone\microphone>flutter pub run pigeon --input D:/creativecreatorormaybenot/microphone/microphone/pigeons/messages.dart
Unhandled exception:
IsolateSpawnException: Unable to spawn isolate: /C:/..AppData/Local/Temp/ec1294b4/_pigeon_temp_.dart:1:8: Error: Error when reading 'd:/creativecreatorormaybenot/microphone/microphone/pigeons/messages.dart': StandardFileSystem only supports file:*
 and data:* URIs
import 'D:\/creativecreatorormaybenot/microphone/microphone/pigeons/messages.dart';
       ^
Instance of '_StringStackTrace'pub finished with exit code 255
```

I am not sure if it is `D:\/` or something else, however, I bet that it is because the import is an absolute path for me.

## Versions

```
pigeon: 0.1.13
Flutter 1.23.0-18.0.pre • channel dev
Dart 2.11.0 (build 2.11.0-213.0.dev)
```

### `beta`

As you can see above, this happened on the `dev` channel. I will try on a different machine running `beta` and test it there as well.

**Comment by creativecreatorormaybenot** [comment]
Tested again on a different Windows machine running `beta`:

```
Flutter 1.23.0-18.1.pre • channel beta
Dart 2.11.0 (build 2.11.0-213.1.beta)
```

I have the same issue there.

**Comment by gaaclarke** [comment]
When we first had this problem i suspected the windows drives might be a problem.  If your source file is on drive D and the temp directory is on drive C I'm not sure that windows can make a relative path that would hop drives and that Dart would recognize.  We can probably just eliminate the usage of the temp directory.  Thanks for the heads up.

**Comment by creativecreatorormaybenot** [comment]
I can confirm that this has been fixed in `0.1.14`.
